modules/multiplayer/scene_cache_interface.cpp

```diff
@@ -244,22 +271,22 @@ bool SceneCacheInterface::send_object_cache(Object *p_obj, int p_peer_id, int &r
 Object *SceneCacheInterface::get_cached_object(int p_from, uint32_t p_cache_id) {
 	Node *root_node = SceneTree::get_singleton()->get_root()->get_node(multiplayer->get_root_path());
 	ERR_FAIL_NULL_V(root_node, nullptr);
-	HashMap<int, PathGetCache>::Iterator E = path_get_cache.find(p_from);
-	ERR_FAIL_COND_V_MSG(!E, nullptr, vformat("No cache found for peer %d.", p_from));
-
-	HashMap<int, PathGetCache::NodeInfo>::Iterator F = E->value.nodes.find(p_cache_id);
-	ERR_FAIL_COND_V_MSG(!F, nullptr, vformat("ID %d not found in cache of peer %d.", p_cache_id, p_from));
+	ERR_FAIL_COND_V(!peers_info.has(p_from), nullptr);
```

Comment:
The old way it was written with `find()` was more optimal, as it avoided double lookup. Is this no longer a concern here?

Reply:
Thanks, fixed!